pkg/supportbundle/rbac.go

```diff
@@ -34,34 +34,75 @@ func checkRemoteCollectorRBAC(ctx context.Context, clientConfig *rest.Config, ti
 
 	var forbidden []error
 
-	spec := authorizationv1.SelfSubjectAccessReviewSpec{
-		ResourceAttributes: &authorizationv1.ResourceAttributes{
-			Namespace:   namespace,
-			Verb:        "create,delete",
-			Group:       "",
-			Version:     "",
-			Resource:    "pods,configmap",
-			Subresource: "",
-			Name:        "",
+	resourceAttributesList := []authorizationv1.ResourceAttributes{
```

Comment:
this could be a constant declared outside so that we will remember to update it in the future.

Reply:
Can you clarify? I'm not sure I follow why moving it from here would make it more visible?

Reply:
I mean setting `resourceAttributesList` as a package constant would make it more discoverable as a configuration rather than hiding it inside `checkRemoteCollectorRBAC`. E.g. when we have new collector with additional permission.

On this, I also think we are missing permission `create pods/exec` for executing `collect` binary.

Reply:
> On this, I also think we are missing permission create pods/exec for executing collect binary.

This PR is for the existing code which I don't think uses `exec` at the moment. 

> I mean setting resourceAttributesList as a package constant would make it more discoverable as a configuration rather than hiding it inside checkRemoteCollectorRBAC. E.g. when we have new collector with additional permission.

I don't have a super strong opinion here, but it doesn't feel like this list is going to be shared across multiple functions, so I'm not sure it needs to come out of this function.